README.md

```diff
@@ -111,6 +100,23 @@ juju remove-relation opensearch self-signed-certificates
 ## Security
 Security issues in the Charmed OpenSearch Operator can be reported through [LaunchPad](https://wiki.ubuntu.com/DebuggingSecurity#How%20to%20File). Please do not file GitHub issues about security issues.
 
+## Customizing
+
```

Comment:
Consider combining this section with the next one into a single section or adding some content between h2 and h3 headings.

Reply:
The main idea is to prevent using one heading directly after another without any content between them, even if the headings are on different levels.

One way to do it is by merging the text in headings, but the suggested heading is a bit too heavy and doesn't feel right. I'd drop the customising part at all. If you think it fits into the Usage section, then it's great. 
And BTW, we should use the British English spelling of customise, as per our [Style guide](https://docs.ubuntu.com/styleguide/en#spelling).